style.css

```diff
@@ -87,3 +92,347 @@ img:hover {
   color: #3c3a39;
   cursor: pointer;
 }
+
+.work-section-heading {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 0 24px 12px 24px;
+  width: 327px;
+}
+
+.work-section-heading h2 {
+  font-family: 'Crete Round', sans-serif;
+  padding: 50px 0 38px 0;
+  text-align: center;
+  font-weight: 500;
+  width: 327px;
+  font-size: 38px;
+  line-height: 48px;
+  color: #1c1a19;
+}
+
+.line {
+  background-color: #b8adad;
+  height: 1px;
+  width: 327px;
+  padding-top: 0;
+  margin-bottom: 120px;
+}
+
+.work-section {
+  display: grid;
+  background: #fff;
+  color: #3c3a39;
+}
+
+.work-section-image {
+  margin: 30px 24px 0 24px;
+}
+
+.img {
+  margin-top: 4px;
+  padding-top: 5px;
+}
+
+.work-section-text1 {
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  margin: 4px 24px;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.paragraph {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 22px;
+  margin: 12px 24px 12px 24px;
+  width: 327px;
+}
+
+.work-section-button {
+  display: flex;
+  width: 302px;
+  gap: 30px;
+  align-items: center;
+}
+
+.btn1 {
+  padding: 10px;
+  margin-bottom: 6px;
+  margin-left: 24px;
+  background-color: #fff;
+  height: 40px;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  text-decoration: none;
+  border: #8993a4 solid 1px;
+}
+
+.btn2 {
+  padding: 10px;
+  margin-bottom: 6px;
+  background-color: #fff;
+  height: 40px;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  text-decoration: none;
+  border: #8993a4 solid 1px;
+}
+
+.btn1:hover {
+  background-color: #ff6b00;
+  color: white;
+}
+
+.btn2:hover {
+  background-color: #ff6b00;
+  color: white;
+}
+
+.btn3 {
+  padding: 10px;
+  margin-bottom: 6px;
+  background-color: #fff;
+  height: 40px;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  text-decoration: none;
+  border: #8993a4 solid 1px;
+}
+
+.btn3:hover {
+  background-color: #ff6b00;
+  color: white;
+}
+
+.btn4 {
+  padding: 5px;
+  margin-bottom: 6px;
+  background-color: #fff;
+  height: 40px;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  text-decoration: none;
+  border: #8993a4 solid 1px;
+}
+
+.btn4:hover {
+  background-color: #ff6b00;
+  color: white;
+}
+
+.project-btn1 {
+  padding: 10px;
+  margin-bottom: 6px;
+  color: #fff;
+  height: 48px;
+  width: 128px;
+  text-align: center;
+  text-decoration: none;
+  border: none;
+  background-color: #ff6b00;
+  margin-left: 24px;
+}
+
+.project-btn1:hover {
+  background-color: #b8adad;
+}
```

Comment:
- [ ] This styling is not according at your project requirements, please take a look at your figma template and make the required changes
![image](https://user-images.githubusercontent.com/52635378/164578495-04c3022d-2358-4dd0-8abf-a59d23eef3cd.png)